tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -95,6 +95,43 @@ class WskBasicTests
             stderr should include(errormsg)
     }
 
+    it should "accept a 3 part Fully Qualified Name without a leading '/'" in {
+        val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+        val guestNamespace = wskprops.namespace
+        val packageName = "packageName3ptFQN"
+        val actionName = "actionName3ptFQN"
+        val triggerName = "triggerName3ptFQN"
+        val ruleName = "ruleName3ptFQN"
+        val fullQualifiedName = s"${guestNamespace}/${packageName}/${actionName}"
+        val validArgs: Seq[Seq[String]] = Seq(
+            Seq("package", "create", packageName),
+            Seq("action", "create", fullQualifiedName, defaultAction.get),
+            Seq("action", "update", fullQualifiedName, defaultAction.get),
+            Seq("action", "invoke", fullQualifiedName),
+            Seq("trigger", "create", triggerName),
+            Seq("action", "get", fullQualifiedName),
+            Seq("rule", "create", ruleName, triggerName, fullQualifiedName),
+            Seq("action", "delete", fullQualifiedName)
+        )
+        val cleanUp: Seq[Seq[String]] = Seq(
+            Seq("package", "delete", packageName),
+            Seq("trigger", "delete", triggerName),
+            Seq("rule", "delete", ruleName)
+        )
+        try {
+            validArgs foreach {
+                cmd =>
+                    val stderr = wsk.cli(cmd ++ wskprops.overrides ++ auth, expectedExitCode = SUCCESS_EXIT).stderr
```

Comment:
Don't need to check standard error here since we are expecting a `SUCCESS_EXIT` code.